public/content.css

```diff
@@ -0,0 +1,41 @@
+
+
+.ghost-text-area {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  max-height: 200px;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  text-align: start;
+  max-height: 200px;
+  overflow-y: hidden;
+  padding: 10px 2.5rem 10px 0.75rem;
+  border-width: 0;
+  margin: 0;
+  background-color: transparent;
+  pointer-events: none;
+  color: transparent;
+}
```

Comment:
This is the main area we are placing ghost text and appending the autocomplete text